src/main/java/Floor.java

```diff
@@ -1,6 +1,6 @@
 public class Floor {
-    double width;
-    double length;
+    private double width;
+    private double length;
 
     public Floor(double width, double length) {
         if (width<0){width=0;}
```

Comment:
1) Please add private access modifiers to Calculator class fields;
2) Remove getWidth() and getLength() methods - you don't need them
3) Simplify getArea() method in 1 row